solara/components/togglebuttons.py

```diff
@@ -133,15 +242,19 @@ def Page():
      * `children`: List of buttons to use as values.
      * `on_value`: Callback to call when the value changes.
      * `dense`: Whether to use a dense (smaller) style.
+     * `mandatory`: Whether selecting at least one element is mandatory.
      * `style`: CSS style to apply to the top level element.
      * `classes`: List of CSS classes to be applied to the top level element.
     """
     class_ = _combine_classes(classes)
     style_flat = solara.util._flatten_style(style)
-    reactive_value = solara.use_reactive(value, on_value)
+    # Similar approach to select.py
```

Comment:
```suggestion
    # See comment in ToggleButtonsSingle regarding the typing issue
```

Is this what you meant?

Reply:
I don't see a comment regarding the typing issue in ToggleButtonsSingle?